config.json

```diff
@@ -8,47 +8,63 @@
     {
       "slug": "hello-world",
       "uuid": "c97d9dda-c7ee-4595-b447-693559d03ba5",
-      "core": false,
+      "core": true,
+      "auto_approve": true,
```

Comment:
since it's hello world, it is not *necessary* to add auto_approve, according to

> We've currently made any hello-world exercises auto-approved in the backend of v2. 

However, in my eyes, it is better to make it explicitly clear that it is auto approved. So I want this line to stay. If anyone disagrees and wants it to be removed, please speak up.

Reply:
The current plan is to eventually remove the code which is auto approving hello-world exercises, and add auto_approve to `config.json` for each track (it's a temporary measure)